libr/core/canal.c

```diff
@@ -2519,12 +2519,29 @@ static int fcn_print_detail(RCore *core, RAnalFunction *fcn) {
 	return 0;
 }
 
+static bool is_fcn_traced(RDebugTrace *traced, RAnalFunction *fcn) {
+	int tag = traced->tag;
+	RListIter *iter;
+	RDebugTracepoint *trace;
+
+	r_list_foreach (traced->traces, iter, trace) {
+		if (!trace->tag || (tag & trace->tag)) {
+			if (r_anal_fcn_in (fcn, trace->addr)) {
+				r_cons_printf ("%d\n", trace->times);
```

Comment:
Should eb acumulati g all the times all the instructions covered by a function has been executed. Also the way you iterate doesnt allows u to identify the function coverage %

Reply:
https://github.com/radare/radare2/pull/12798#discussion_r248486907

should fcn coverage be implemented in this pr only ?

>Also the way you iterate doesnt allows u to identify the function coverage %

the above code will be slightly modified for storing total size (traced_fcn_size) of all the instructions of function being traced. then we will take (traced_fcn_size / fcn_size) to show coverage %. Is this approach good ?